services/api-gateway/src/simcore_service_api_gateway/endpoints_user.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter, Depends, Security
+
+from .auth import get_current_active_user, get_current_user
+from .schemas import User
+
+router = APIRouter()
+
+
+@router.get("/users/me/", response_model=User)
+async def read_users_me(current_user: User = Depends(get_current_active_user)):
+    return current_user
+
+
+@router.get("/users/me/projects/")
+async def list_own_projects(
+    current_user: User = Security(get_current_active_user, scopes=["projects"])
```

Comment:
So that means that the user needs the scope "me" and "projects". But where does the "me" come frome?

Reply:
Ah never mind. Got it. But why do you need once a Depends(get_current_user) and then a Security(get_current_user)